app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -1365,7 +1365,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputName_inputPin_inputConfirmPin_deselectPIN_clickCreate_checkOpensProfileActivity() { // ktlint-disable max-line-length
+  fun testAddProfile_inputName_Pin_inputConfirmPin_deselectPIN_clickCreate_OpensProfileActivity() {
```

Comment:
```suggestion
  fun testAddProfile_inputName_Pin_inputConfirmPin_deselectPIN_clickCreate_opensProfileActivity() {
```

(Be sure to take care on the capitalization of test names).